test/LocalizationWebsite/StartupResourcesInClassLibrary.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Localization;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+using Microsoft.Extensions.Logging;
+
+namespace LocalizationWebsite
+{
+    public class StartupResourcesInClassLibrary
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddLocalization(options => options.ResourcesPath = "Resources");
```

Comment:
We do not, which I know because it was broken. I added Create_FromType_ResourcesPathDirectorySeperatorToDot and Create_FromNameLocation_ResourcesPathDirectorySeperatorToDot to cover this area.